src/OrchardCore.Modules/OrchardCore.ContentTypes/Controllers/AdminController.cs

```diff
@@ -172,6 +172,12 @@ public async Task<ActionResult> Edit(string id)
     public async Task<ActionResult> EditPost(
         string id, EditTypeViewModel viewModel, [Bind(Prefix = "submit.Save")] string submitSave)
     {
+        // if the submitSave is null, it means the user clicked on the "Delete" button
+        if (submitSave is null)
+        {
+            return await Delete(id);
```

Comment:
Why did you change the implementation, just add the functionality back as it was. Deleting from the edit method doesn't make sense.

Reply:
In the past, there was a single place for the delete button, which was triggered from within the form.

Reply:
I will try to change the default action name in the form

Reply:
I see, it had these on the action:

```csharp
[HttpPost, ActionName("Edit")]
[FormValueRequired("submit.Delete")]
public async Task<ActionResult> Delete(string id)
```

You're right, this can be improved.

Reply:
Just move the delete button out of the edit form and make it stand-alone like in the list view.

Reply:
This is an easy choice, but the design needs to be tweaked